css-gaps-1/Overview.bs

```diff
@@ -482,99 +482,217 @@ Pairing gap intersection points into segments</h4>
 		</figure>
 	</div>
 
-<h3 id="outset">
-Adjusting gap decoration endpoints: The 'column-rule-outset', 'row-rule-outset', and 'rule-outset' properties</h3>
+<h3 id="inset">
+Adjusting gap decoration endpoints: The 'rule-inset' properties</h3>
 
 	<pre class='propdef'>
-		Name: column-rule-outset, row-rule-outset
-		Value: <<length-percentage>>
+		Name: column-rule-edge-start-inset, column-rule-edge-end-inset, row-rule-edge-start-inset, row-rule-edge-end-inset
+		Value: auto | <<length-percentage>>
+		Initial: ''0''
```

Comment:
Initial value should be `auto` for both sets of properties - that will make shorthanding them cleaner.